src/NuGetGallery.Core/GitHub/RepositoryInformation.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class RepositoryInformation
+    {
+        public RepositoryInformation()
```

Comment:
Please remove if nothing uses this

Reply:
I think I had to add it or the Deserialization failed. I would have to test it again

Reply:
Newtonsoft.Json should use the constructor that accepts parameters unless there's mismatches in names or types

Reply:
I'd like to get rid of the `set;` methods on properties and the ` = null;` on `Dependencies`